reference/docs/conversion/sakai_12_oracle_conversion.sql

```diff
@@ -372,3 +372,31 @@ ALTER TABLE SAM_QUESTIONPOOL_T MODIFY DESCRIPTION CLOB;
 UPDATE SAM_QUESTIONPOOL_T SET DESCRIPTION = DESCRIPTION_COPY;
 
 ALTER TABLE SAM_QUESTIONPOOL_T DROP COLUMN DESCRIPTION_COPY;
+
+-- SAK-30461 Portal bullhorns
+CREATE TABLE BULLHORN_ALERTS
+(
+    ID NUMBER(19) NOT NULL,
+    ALERT_TYPE VARCHAR(8) NOT NULL,
+    FROM_USER VARCHAR2(99) NOT NULL,
+    TO_USER VARCHAR2(99) NOT NULL,
+    EVENT VARCHAR2(32) NOT NULL,
+    REF VARCHAR2(255) NOT NULL,
+    TITLE VARCHAR2(255),
+    SITE_ID VARCHAR2(99),
+    URL CLOB NOT NULL,
+    EVENT_DATE TIMESTAMP NOT NULL,
+    PRIMARY KEY(ID)
+);
+
+CREATE SEQUENCE bullhorn_alerts_seq;
+
+BEGIN
+EXECUTE IMMEDIATE
```

Comment:
You don't need to do this little trick just add the real sentence:
```
CREATE OR REPLACE TRIGGER bullhorn_alerts_bir
 BEFORE INSERT ON BULLHORN_ALERTS
 FOR EACH ROW
 BEGIN
      SELECT bullhorn_alerts_seq.NEXTVAL
      INTO   :new.id
      FROM   dual; 
END;
/
```

Reply:
I noticed that the trailing forward slash after `END;` didn't make it into the conversion script. Is this bit necessary? We ran this as is (without the slash), and it ended up creating the trigger in an invalid format, and thus wouldn't compile, leading to stack traces. When I inspected the trigger, this is how it ended up being created:

```
create or replace TRIGGER bullhorn_alerts_bir
    BEFORE INSERT ON BULLHORN_ALERTS
    FOR EACH ROW
    BEGIN
        SELECT bullhorn_alerts_seq.NEXTVAL
        INTO   :new.id
        FROM   dual
```

Notice the absence of the semicolon after `FROM dual`, and the missing `END;` on the following line. Could this be explained by the missing forward slash in comment above?

cc: @adrianfish, @jmarianolujan

Reply:
I also found this article: https://stackoverflow.com/questions/1079949/when-do-i-need-to-use-a-semicolon-vs-a-slash-in-oracle-sql, which states:

> The / is mainly required in order to run statements that have embedded ; like a CREATE PROCEDURE statement.

Reply:
We've confirmed locally that the trailing slash is necessary. Without it, the script fails.